docs/samples/calling/app.js

```diff
@@ -233,6 +282,7 @@ async function initCalling(e) {
       country: country.value,
     },
     serviceData,
+    jwe: jwtTokenForDestElm.value,
```

Comment:
Are we manually populating the JWT here?

Reply:
Added a button to generate the guest access token